funnel/templates/macros.html.jinja2

```diff
@@ -744,6 +744,13 @@
           <h2 class="mui--text-headline mui--text-bold zero-bottom-margin" data-cy="profile-title"><a href="{{ profile.url_for() }}" class="mui--text-dark">{{ profile.title }}</a></h2>
           <p class="mui--text-title mui--text-light">@{{ profile.name }}</p>
         </div>
+        <p>
+          <span class="mui--visible-xs-block mui--visible-sm-block mui--visible-md-inline-block mui--visible-lg-inline-block mui--visible-xl-inline-block">{{ faicon(icon='history', icon_size='caption') }} {% trans date=profile.owner.created_at|date%}Joined {{ date }}{% endtrans %}</span>
```

Comment:
We want `MMM YYYY`. No day. Partly because day will be ±1 depending on viewer's timezone.